src/invertObject.js

```diff
@@ -19,7 +19,17 @@
  * @return {Object}
  */
 function invertObject(items) {
-  // write code here
+  const resultObject = {};
+
+  for (const key in items) {
+    if (!resultObject.hasOwnProperty(items[key])) {
```

Comment:
```suggestion
    if (resultObject.hasOwnProperty(items[key])) {
      return null;
    }
```
это позволит избавиться от else